### ISSUE TITLE
gatsby develop fails -  reserved field name in the context object

### ISSUE THREAD

**Issue by samOpenforce** [microapps/gatsby-plugin-react-i18next#13]
Hi, trying to use this plugin but receive the following error. Seems to be a conflict with Graphql. Any ideas for a workaround/override? Any suggestions gratefully received!

```
 ERROR #11324 

The plugin "gatsby-plugin-react-i18next" used a reserved field name in the context object when creating a page:

  * "path"

{
   ...myObject...
}

Data in "context" is passed to GraphQL as potential arguments when running the
page query.

When arguments for GraphQL are constructed, the context object is combined with
the page object so *both* page object and context data are available as
arguments. So you don't need to add the page "path" to the context as it's
already available in GraphQL. If a context field duplicates a field already
used by the page object, this can break functionality within Gatsby so must be
avoided.

Please choose another name for the conflicting fields.

The following fields are used by the page object and should be avoided.

  * "path"
  * "matchPath"
  * "component"
  * "componentChunkName"
  * "pluginCreator___NODE"
  * "pluginCreatorId"



See the documentation for the "createPage" action — https://www.gatsbyjs.org/docs/actions/#createPage
```

**Comment by cn2amz** [comment]
The compilation was ok before, and the problem has been reproduced since today. It may be caused by the latest version upgrade. error message as below:


ERROR #11324

The plugin "gatsby-plugin-react-i18next" used a reserved field name in the context object when creating a page:

  * "path"

{
    "path": "/404.html",
    "component": "//src/pages/404.js",
    "context": {
        "path": "/404",
        "language": "en",
        "i18n": {
            "language": "en",
            "languages": [
                "en"
            ],
            "defaultLanguage": "en",
            "generateDefaultLanguagePage": false,
            "routed": false,
            "originalPath": "/404",
            "path": "/404"
        }
    },
    "status": "404"
}

Data in "context" is passed to GraphQL as potential arguments when running the
page query.

When arguments for GraphQL are constructed, the context object is combined with
the page object so *both* page object and context data are available as
arguments. So you don't need to add the page "path" to the context as it's
already available in GraphQL. If a context field duplicates a field already
used by the page object, this can break functionality within Gatsby so must be
avoided.

Please choose another name for the conflicting fields.

The following fields are used by the page object and should be avoided.

  * "path"
  * "matchPath"
  * "component"
  * "componentChunkName"
  * "pluginCreator___NODE"
  * "pluginCreatorId"



See the documentation for the "createPage" action — https://www.gatsbyjs.com/docs/reference/config-files/actions#createPage

not finished createPagesStatefully - 0.465s

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: C:\Users\charlie\AppData\Local\nvs\node\18.2.0\x64\node.exe

**Comment by matiaskorhonen** [comment]
I'm getting this same issue with a brand new Gatsby site (Gatsby 4.20.0, gatsby-plugin-react-i18next 2.0.3).

**Comment by liuuuk311** [comment]
Same error here!

**Comment by goslingo** [comment]
Same error for me too... using Gatsby 4.18.0, and gatsby-plugin-react-i18next 2.0.1

**Comment by jimmyn** [comment]
It is fixed in 2.0.4
